### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by mdalmazan**
This change is reference to #67 where the logo is swapped out.

### CODE REVIEW

**Comment by mdalmazan** [comment]
Second commit is reference to #66

**Comment by jridener** [comment]
@mdalmazan Thanks for the pull request! We're working on a quick review and will merge soon.

File: wic/src/components/SurveyResults.js [hunk 1]
```diff
@@ -36,21 +36,16 @@ class SurveyResults extends React.Component {
...
-              <div>Email Food Connection at foodconnection@shfb.org.</div>
-            </p>
-            <p>
-              <div>Text "Web Connect" at 408-455-5181</div>
-            </p>
+            <br />
+            <p>Contact Second Harvest Food Bank:</p>
+            <ul>
+              <li>Visit their  <a href="SHFB.org/getfood">website.</a></li>
+              <li>Call <a href="+1-800-984-3663">1-800-984-3663</a></li>
```
Comment (RohitTalwalkar) [inline wic/src/components/SurveyResults.js]: Turns out you can also make the link work as a phone number if you prefix it `tel:`
http://www.w3.org/TR/mwabp/#bp-interaction-uri-schemes

File: wic/src/components/SurveyResults.js [hunk 2]
```diff
@@ -36,21 +36,16 @@ class SurveyResults extends React.Component {
...
-            <p>            
-              <div>Email Food Connection at foodconnection@shfb.org.</div>
-            </p>
-            <p>
-              <div>Text "Web Connect" at 408-455-5181</div>
-            </p>
+            <br />
+            <p>Contact Second Harvest Food Bank:</p>
+            <ul>
+              <li>Visit their  <a href="SHFB.org/getfood">website.</a></li>
```
Comment (RohitTalwalkar) [inline wic/src/components/SurveyResults.js]: nit: extra space.
Also, I think you'd want the href to start with `https://`

File: wic/src/components/SurveyResults.js (hunk 1 above)
Comment (RohitTalwalkar) [reply wic/src/components/SurveyResults.js]: @jridener - did we also want to put the hours next to the phone number?
 (Monday - Friday, 8 a.m.-5 p.m.)

**Pull request merged**